android/app/src/main/assets/00_NotoNaskhArabic-Regular.ttf

```diff
@@ -0,0 +1 @@
+../../../../../data/00_NotoNaskhArabic-Regular.ttf
```

Comment:
I moved android/app/src/assets to android/app/src/main/assets to match the default Android Studio layout. This folder is picked up by gradle automatically without any hacks with assets.srcDirs = ["assets"].